src/analysis/asymptotics/asymptotics.lean

```diff
@@ -1247,6 +1247,33 @@ end exists_mul_eq
 
 /-! ### Miscellanous lemmas -/
 
+theorem is_O_of_div_is_bounded_under {α : Type*} {l : filter α}
+  {f g : α → 𝕜} (hgf : ∀ᶠ x in l, g x = 0 → f x = 0)
+  (h : is_bounded_under (≤) l (λ x, ∥f x / g x∥)) :
+  is_O f g l :=
+begin
+  obtain ⟨c, hc⟩ := h,
+  rw filter.eventually_iff at hgf hc,
+  simp only [mem_set_of_eq, mem_map, normed_field.norm_div] at hc,
+  refine is_O_iff.mpr ⟨c, filter.eventually_of_mem (inter_mem_sets hgf hc) (λ x hx, _)⟩,
+  by_cases hgx : g x = 0,
+  { simp [hx.1 hgx, hgx] },
+  { refine (div_le_iff (norm_pos_iff.mpr hgx)).mp hx.2 },
+end
+
+lemma is_bounded_under_of_tends_to_nhds {α : Type*} {l : filter α}
+  {f : α → 𝕜} (c : 𝕜) (h : filter.tendsto f l (𝓝 c)) :
+  is_bounded_under (≤) l (λ x, ∥f x∥) :=
```

Comment:
I should have seen that earlier, but I think we always write `tendsto` instead of`tends_to` to refer to the definition name. Also, I think this lemma should go somewhere else, but I'm not sure where (maybe `analysis.normed_space.basic` ?), so maybe we should ask a maintainer about that.

Reply:
I just put this in that file for now. There are other similar lemmas with similar hypothesis to group it with, so I think that it makes sense to have it there